openedx/core/djangoapps/course_date_signals/tests.py

```diff
@@ -114,5 +116,261 @@ def test_sequence_with_graded_and_ungraded_assignments(self):
                 (ungraded_problem_2.location, {'due': None}),
                 (graded_problem_1.location, {'due': 5}),
             ]
-            sequence = modulestore().get_item(sequence.location)
             self.assertCountEqual(_gather_graded_items(sequence, 5), expected_graded_items)
+
+    def test_get_custom_pacing_children(self):
+        """
+        _get_custom_pacing_items should return a list of (block item location, field metadata dictionary)
+        where the due dates are set from due_num_weeks
+        """
+        # A subsection with multiple units but no problems
+        with modulestore().bulk_operations(self.course.id):
+            sequence = ItemFactory(parent=self.course, category='sequential', due_num_weeks=2)
+            vertical1 = ItemFactory(parent=sequence, category='vertical')
+            vertical2 = ItemFactory(parent=sequence, category='vertical')
+            vertical3 = ItemFactory(parent=sequence, category='vertical')
+            expected_dates = [
+                (sequence.location, {'due': timedelta(weeks=2)}),
+                (vertical1.location, {'due': timedelta(weeks=2)}),
+                (vertical2.location, {'due': timedelta(weeks=2)}),
+                (vertical3.location, {'due': timedelta(weeks=2)})
+            ]
+            self.assertCountEqual(_get_custom_pacing_children(sequence, 2), expected_dates)
+
+        # A subsection with multiple units, each of which has a problem
```

Comment:
nit: Indent your comments in this test and add in what you're testing for. As in
Problems should also inherit due date
ORA should not receive due date